api/Permissions.json

```diff
@@ -637,6 +637,45 @@
           }
         }
       },
+      "permission_screen-wake-lock": {
```

Comment:
Found this Permission API permission in the spec - we somehow missed it last time.

Tested with on the indicated versions (and closest I can find before). These are supported (or would otherwise have thrown in test)
```
(async () => {
    try {
        const result = await navigator.permissions.query({ name: "screen-wake-lock" });
        console.log(result);
    } catch (error) {
        console.error(error);
    }
})();
```